modules/sandboxed-containers-deleting-operator.adoc

```diff
@@ -0,0 +1,37 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-operator_{context}"]
+= Deleting {sandboxed-containers-operator}
+
+You can delete the {sandboxed-containers-operator} from your cluster.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Delete the {sandboxed-containers-operator} subscription from Operator Lifecyle Manager (OLM) by running the following command:
++
+[source,terminal]
+----
+$ oc delete subscription openshift-sandboxed-containers-subscription -n openshift-sandboxed-containers-operator
+----
+
+. Set the cluster service version (CSV) name for {sandboxed-containers-first} as an environment variable by running the following command:
++
+[source,terminal]
+----
+CSV_NAME=$(oc get csv -n openshift-sandboxed-containers-operator -o=custom-columns=:metadata.name)
+----
+
+. Delete the CSV name for {sandboxed-containers-first}.
```

Comment:
Two items:

* I prefer omitting "by running the following command" in all cases, but the preceding two include it.
* End with a colon, I'm pretty sure.